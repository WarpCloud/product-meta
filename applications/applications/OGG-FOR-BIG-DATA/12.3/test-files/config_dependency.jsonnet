{
  application_type: "OGG-FOR-BIG-DATA",
  application_name: "ogg-for-big-data",
  application_version: "12.3",
  user_config: {},
}
