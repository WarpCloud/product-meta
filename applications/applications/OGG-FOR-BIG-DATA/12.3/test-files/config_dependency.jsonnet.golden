{
   "Customized_Namespace": "default",
   "Develop": false,
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "ogg-for-big-data",
            "application_type": "OGG-FOR-BIG-DATA"
         },
         "configs": { },
         "instance_settings": { },
         "moduleName": "ogg-for-big-data",
         "name": "ogg-for-big-data-ogg-for-big-data",
         "version": "12.3"
      }
   ],
   "instance_settings": { },
   "version": ""
}
